corehq/apps/analytics/tasks.py

```diff
@@ -368,7 +369,8 @@ def track_periodic_data():
     submit = []
     for user in users_to_domains:
         email = user.get('email')
-        if not email:
+        if not email or not validate_email(email, verify=True):
```

Comment:
One thing to note about `validate_email` is that it returns `None` if it's unsure. so this is being extra stringent. We can change this so it only skips when we now it's not valid